src/app/containers/InlineLink/index.new.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import InlineLink from '../../components/InlineLink';
+import Blocks from '../Blocks';
+import fragment from '../Fragment';
```

Comment:
Should this be capitalised? EG: `Fragment`

Reply:
It could be, but importing it as lowercase enables a shorthand for [componentsToRender](https://github.com/BBC-News/simorgh/pull/780/files#diff-cb00553c1a2942e5f1497e226e6f9db1R10), and is a pattern we've used elsewhere.